sdk/options.go

```diff
@@ -50,6 +52,9 @@ type Options struct {
 	// inmem store.
 	Store storage.Store
 
+	// Hooks allows hooking into the internals of SDK operations (TODO(sr): find better words)
```

Comment:
```suggestion
	// Hooks allows instrumenting and mutating the internals of SDK operations and data structures.
```
[suggestion]

Reply:
I missed the "allows" -> "allow" transition that needed to happen. :facepalm: Sorry about that!